locations/spiders/bank_of_hawaii_us.py

```diff
@@ -0,0 +1,37 @@
+import scrapy
+from scrapy.http import JsonRequest
+
+from locations.categories import Categories, Extras, apply_category, apply_yes_no
+from locations.dict_parser import DictParser
+from locations.hours import OpeningHours
+
+
+class BankOfHawaiiUSSpider(scrapy.Spider):
+    name = "bank_of_hawaii_us"
+    item_attributes = {"brand": "Bank of Hawaii", "brand_wikidata": "Q4856124"}
+    allowed_domains = ["boh.com"]
+    start_urls = ["https://www.boh.com/bohapi/get-locations?lat=21.33&lng=-157.845934&radius=250"]
+
+    def parse(self, response):
+        for location in response.json()["locations"]:
+            item = DictParser.parse(location)
+            item["lat"] = location["geocode"]["latitude"]
+            item["lon"] = location["geocode"]["longitude"]
+            oh = OpeningHours()
+            for day_name, day in location["operationalHours"]["hours"].items():
+                if day["isClosed"] == True:
+                    continue
+                oh.add_range(day_name.title(), day["hours"][0]["openTime"], day["hours"][0]["closeTime"], "%H:%M:%S")
```

Comment:
Is there ever `day["hours"][1]`? Should we loop it?

Reply:
I haven't seen multiple hours ranges, but it doesn't hurt to add the logic for it so I've fixed it :)